lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -642,6 +642,12 @@ class BarTouchTooltipData with EquatableMixin {
   /// The tooltip background color.
   final Color tooltipBgColor;
 
+  /// The tooltip border color.
+  final Color tooltipBorderColor;
```

Comment:
I think it's better to have a [BorderSide](https://api.flutter.dev/flutter/painting/BorderSide-class.html) (which contains `color` and `width`) Instead of defining two variables.

Reply:
good idea